understanding/22/focus-not-obscured-minimum.html

```diff
@@ -41,12 +41,30 @@ <h2>Intent of Focus Not Obscured</h2>
        </p>
 
     </section>
+       <section id="Considerations">
+         <h3>Considerations</h3>
+         The following sub-sections provide clarity on how to assess Focus Not Obscured (Minimum) in certain scenarios.
+         <section><h4>User-movable</h4>
+         <p>This SC contains a note regarding content that can be repositioned. If users can move content regions, then they can potentially position them such that they obscure other content that may receive focus. In such a case, the author is only responsible for ensuring that the movable content <em>in its initial position</em> does not obscure the item receiving focus.</p>
+         <p>This note is intended to accommodate a common interaction in complex applications such as authoring tools, where the main editing region (also called a canvas) can be enhanced by displaying toolbars or other panels, which can be repositioned around the canvas. It is possible to design such toolbars so they do <em>not</em> obscure focus. Authors are encouraged to do so, as well as pursue techniques which ensure equitable keyboard use of such toolbars. However, in recognition of the complexities involved in responsive design as well as in supporting the ability to transform the text size and spacing of content, only the starting position of such side panels is assessed.</p>
+     
+       </section>
+       <section>
+         <h4>Modal dialogs versus overlays</h4>
+         <p>A properly constructed modal dialog will always pass this SC. Even if it appears directly on top of an item with focus, the dialog takes focus on appearance, and thus the item receiving focus -- the dialog or one of its components -- is visible. A properly constructed modal maintains that focus and prevents interaction outside the modal until it is dismissed.</p>
+         <p>A dialog-like overlay that does not take focus on appearance and does not either constrain interaction to the overlay or dismiss itself on loss of focus (thus allowing focus to exit into the content behind it) will be at risk of failing this SC, where it is positioned such that it can obscure other content.</p>
```

Comment:
```suggestion
         <p>A dialog-like overlay that does not take focus on appearance and does not either constrain interaction to the overlay or dismiss itself on loss of focus (thus allowing focus to exit into the content behind it) will be at risk of failing this SC, where it is positioned such that it can obscure other focusable items.</p>
```